documentation/modules/auxiliary/admin/http/atlassian_confluence_auth_bypass.md

```diff
@@ -0,0 +1,45 @@
+# Description
+
+This module exploits an Broken Access Control vulnerability in Atlassian Confluence servers leads to Authentication Bypass.
+
+A specially crafted request can be create new admin account without authorization in the Atlassian server.
+## Vulnerable Application
+Affecting Atlassian Confluence from version 8.0.0 to before 8.3.3, from version 8.4.0 before 8.4.3 and from version 8.5.0 before 8.5.2.
+
+## Verification Steps
+
+- [ ] Setting up a working installation of Atlassian Confluence Server before 8.0.0
```

Comment:
change these to being numbered

Reply:
Okay, I changed it via 49e736f5e8316217d6f58f042eae7bbd23d5a710